pkg/spec/storage.go

```diff
@@ -61,6 +66,8 @@ func (config *CreateConfig) parseVolumes(runtime *libpod.Runtime) ([]spec.Mount,
 	// Unify --mount, --volume, --tmpfs mounts
 	unifiedMounts := make(map[string]spec.Mount)
 
+	// Add image volumes
```

Comment:
Are you missing part of the patch set?

Reply:
I think you're on an older version of the patch - the rest is addressed in the latest commit

Reply:
Yes I think I was reviewing this when you force pushed.  
Anyways no way to do a great job on reviewing this, so hope tests catch any failures.